src/CommunityToolkit.Maui.SourceGenerators/Generators/TextColorToGenerator.cs

```diff
@@ -18,97 +26,80 @@ class TextColorToGenerator : IIncrementalGenerator
 
 	public void Initialize(IncrementalGeneratorInitializationContext context)
 	{
+		if (!Debugger.IsAttached)
```

Comment:
Please remove

Reply:
This is an old way to debug Source Generators that's no longer required.

Here's how to debug Source Generators using a Debug Launch UI Profile in VS 2022: https://github.com/JoanComasFdz/dotnet-how-to-debug-source-generator-vs2022

As a demonstration, here's the debugger hitting a break point in the Source Generator without the `Debugger.Launch()` logic:

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/13558917/206589935-0210c115-6d13-471c-8529-0077ecd4533b.png">